src/main/java/models/Ticket.java

```diff
@@ -88,22 +84,33 @@ private boolean controlEventCode(short eventCode) {
         return true;
     }
 
-    private boolean controlInformationAboutSector(Sector sector){
-        if(sector != Sector.A & sector != Sector.B & sector != Sector.C){
+    private boolean controlInformationAboutSector(Sector sector) {
+        if (sector != Sector.A & sector != Sector.B & sector != Sector.C) {
             System.out.println("We don't have this sector");
             return false;
         }
         return true;
     }
 
-    private boolean controlLengthNamePlace(String namePlace){
-        if (namePlace.length() > 10){
+
+
+    private boolean controlLengthNamePlace(String namePlace) {
+        if (namePlace.length() > 10) {
             System.out.println("Name place have more 10 symbols");
             return false;
         }
         return true;
     }
 
+    public void typeDeviceAccordingEventCode(Ticket ticket) {
+        System.out.println("If your code event <333 you visit club and you can use only number of your phone\n" +
+                           "If your code event >333 and <667 you visit theater. We send ticket on your email or you" +
+                "can use number of your phone\n" +
+                           "If your code event  >=667 you visit cinema. We send ticket on your email or you" +
+                "can use number of your phone\n"
+        );
+    }
+
     @Override
     public String toString() {
```

Comment:
follow the order:
constants, variables, conctructors, setters/getters, equals, hashcode, toString, other methods

Reply:
+